spec/lib/license_finder/package_manager_spec.rb

```diff
@@ -73,59 +73,59 @@ module LicenseFinder
 
     describe '#prepare' do
       context 'when there is a prepare_command' do
+        let(:prepare_error) { /Prepare command .* failed/ }
+        let(:project_path) { '/path/to/project' }
+        let(:log_directory) { '/path/to/project/logs/project' }
+        let(:log_file_path) { File.join(log_directory, 'errors.log') }
```

Comment:
we should add test to cover existence of .package_management_command

Reply:
Sure thing.